.gitignore

```diff
@@ -1,12 +1,3 @@
 .DS_Store
-.yarn/*
-node_modules/
```

Comment:
please keep the `.gitignore` as it is or at least the list of dependency folders, in addition to your `.env.*` change.

Reply:
ah, right. no deps, no folder 💯. 

then maybe only keep the `.yarn` one. afaik v2 creates the folder even though no deps, just to keep the project specific version in it.